borgmatic/hooks/mongodb.py

```diff
@@ -161,4 +161,7 @@ def build_restore_command(extract_process, database, dump_filename):
         command.extend(('--authenticationDatabase', database['authentication_database']))
     if 'restore_options' in database:
         command.extend(database['restore_options'].split(' '))
+    if database['schemas']:
+        for schema in database['schemas']:
+            command.extend(('--nsInclude', schema))
```

Comment:
Would it be possible to add unit tests for the `schemas` case with these MongoDB and PostgreSQL functions? In the case of PostgreSQL, the simplest thing would be to add another test for `restore_database_dump()` with appropriate mocking. An alternative though would to be to break out a function that builds the restore command and then test _that_. However, that would require more changes to existing tests. So, a trade-off either way.